.github/workflows/run-forge-tests.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
         run: forge install
 
       - name: Run tests
-        run: forge test -vvv
+        run: forge test -vvv --via-ir
```

Comment:
This PR has been hitting the following failure when forge compiles the contracts:

<img width="1516" alt="image" src="https://user-images.githubusercontent.com/2570291/202321977-a42c3ea3-4c32-4e44-9422-54244111fe1f.png">

```
CompilerError: Stack too deep. Try compiling with `--via-ir` (cli) or the equivalent `viaIR: true` (standard JSON) while enabling the optimizer. Otherwise, try removing local variables. When compiling inline assembly: Variable headStart is 1 slot(s) too deep inside the stack.
```

Compiling with `--via-ir` seems to work fine, though.